OutOfSchool/OutOfSchool.BusinessLogic/Util/Mapping/ExternalExportMappingProfile.cs

```diff
@@ -79,7 +79,7 @@ public ExternalExportMappingProfile()
 
         CreateMap<InstitutionHierarchy, SubDirectionsInfoDto>()
```

Comment:
As comment in the export service - we can query the subdirections repository directly, no need to get hierarchies.